### PR TITLE
howdoi: vendor Python deps

### PR DESCRIPTION
**Pull request by timsutton**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/h/howdoi.rb [hunk 1]
```diff
@@ -22,28 +22,26 @@ class Howdoi < Formula
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "rust" => :build
-  depends_on "cffi"
```
[the remark this answers is not in the document]
Comment (timsutton) [reply Formula/h/howdoi.rb:25]: `cffi` looks like it's a downstream dependency of `keep`:
```
Collecting cffi>=1.4.1 (from pynacl>=1.4.0->PyGithub->keep==2.9->-r requirements/common.txt (line 10))
```

[the remark this answers is not in the document]
Comment (timsutton) [reply Formula/h/howdoi.rb:25]: I had to manually add `cffi` and `pycparser` resources (the latter is a dependency of `cffi`)

[the remark this answers is not in the document]
Comment (cho-m) [reply Formula/h/howdoi.rb:25]: `cffi` and `pycparser` are getting removed from resources due to `cryptography`.

Still need to figure out that situation if we want to keep the `cryptography` formula.

[the remark this answers is not in the document]
Comment (p-linnane) [reply Formula/h/howdoi.rb:25]: It was decided we will keep the `cryptography` formula to reduce Rust build deps all over the place.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**